### PR TITLE
Google Analytics _set*SampleRate and _set*CookieTimeout

### PR DESCRIPTION
**Pull request by saschwarz**
I added support for GA's `_setSampleRate` and `_setSiteSpeedSampleRate` settings for my needs and then saw `_setSessionCookieTimeout` and `_setVisitorCookieTimeout` would be easy to add too.

I've updated the tests and the docs and thought it might be of interest to other developers.
Thanks for this project!
Steve

### CODE REVIEW

**Comment by bittner** [comment]
Hi @saschwarz, thanks for your contribution! Just wanted to say, don't worry about the failing build: That's just for Python 3.2, and will go away once #86 is merged.

File: analytical/templatetags/google_analytics.py [hunk 1]
```diff
@@ -131,8 +139,37 @@ def _get_other_commands(self, context):
...
         if getattr(settings, 'GOOGLE_ANALYTICS_SITE_SPEED', False):
             commands.append(SITE_SPEED_CODE)
+
         if getattr(settings, 'GOOGLE_ANALYTICS_ANONYMIZE_IP', False):
             commands.append(ANONYMIZE_IP_CODE)
+
+        sampleRate = getattr(settings, 'GOOGLE_ANALYTICS_SET_SAMPLE_RATE', False)
+        if sampleRate is not False:
+            value = decimal.Decimal(sampleRate)
+            if value < 0 or value > 100:
```
Comment (bittner) [inline analytical/templatetags/google_analytics.py]: Could be written as:
``` python
            if not 0 <= value <= 100:
```

File: analytical/templatetags/google_analytics.py [hunk 2]
```diff
@@ -131,8 +139,37 @@ def _get_other_commands(self, context):
...
+                raise AnalyticalException("'GOOGLE_ANALYTICS_SET_SAMPLE_RATE' must be >= 0 and <= 100")
+            commands.append(SET_SAMPLE_RATE_CODE % value.quantize(TWOPLACES))
+
+        siteSpeedSampleRate = getattr(settings, 'GOOGLE_ANALYTICS_SET_SITE_SPEED_SAMPLE_RATE', False)
+        if siteSpeedSampleRate is not False:
+            value = decimal.Decimal(siteSpeedSampleRate)
+            if value < 0 or value > 100:
```
Comment (bittner) [inline analytical/templatetags/google_analytics.py]: Could be written as:
``` python
            if not 0 <= value <= 100:
```

**Comment by jcassee** [comment]
I have a question: why do the settings have `_SET_` in them? I would have expected names like `GOOGLE_ANALYTICS_SAMPLE_RATE`. Do you feel it adds something?

**Comment by saschwarz** [comment]
@jcassee I was just "underscoring" the names GA uses in their variable names: '_setSampleRate' -> 'SET_SAMPLE_RATE'. But you've made me realize I didn't follow the precedent set in the existing tags. I can update the PR if you like.

**Comment by bittner** [comment]
@saschwarz If you merge `master` into your feature branch (or rebase to HEAD) the build will fully pass.

**Comment by jcassee** [comment]
If possible I would prefer rebasing.

**Comment by saschwarz** [comment]
@jcassee would you like the settings renamed? I can do that tonight and then squash/rebase.

**Comment by jcassee** [comment]
Yeah, unless you have an argument against. It's four characters less to type. 😄

**Comment by saschwarz** [comment]
The majority begin with `_set` . Notable exceptions appear to be for ecommerce/event tracking but those are more likely to be invoked dynamically by JS on the client.

**Comment by bittner** [comment]
I also vote for removing `SET_` from the constants. Just because Google engineers write questionable code this doesn't mean we should forget [PEP 20](http://legacy.python.org/dev/peps/pep-0020/), the Zen of Python. Let's put readable code over compliance with Google. _"Beautiful is better than ugly"_

**Pull request merged**